README.md

```diff
@@ -22,16 +22,6 @@ This plugin interfaces with the Splunk HTTP Event Collector:
       check_index false
       source {TAG}
       sourcetype fluent
-
-      # TIMESTAMP: key1="value1" key2="value2" ...
-      time_format unixtime
-      format kvp
-
-      # Memory buffer with a short flush internal.
```

Comment:
I was specifically referencing the buffer parameters, that's why I placed my comment on L30